README.md

```diff
@@ -4,13 +4,174 @@
 * [Introduction](#introduction)
 * [Requirements](#requirements)
 
-## Introduction
+# Introduction
 
-Sciuro is a bridge between alertmanager and Kubernetes to sync alerts as
-Node Conditions. It is designed to work in tandem with other controllers
-that observe Node Conditions such as [draino](https://github.com/planetlabs/draino) or the [cluster-api](https://cluster-api.sigs.k8s.io/tasks/healthcheck.html).
+Sciuro is a bridge between alertmanager and Kubernetes to sync alerts as Node
+Conditions. It is designed to work in tandem with other controllers that
+observe Node Conditions such as [draino](https://github.com/planetlabs/draino)
+or the [cluster-api](https://cluster-api.sigs.k8s.io/tasks/healthcheck.html).
 
-## Requirements
+# Requirements
 
 * Alertmanager API v2
 * Kubernetes 1.12+
+
+# Deployment
+1. Download the manifests from the latest Github release
+
+```
+wget https://github.com/cloudflare/sciuro/releases/download/v0.1.0/cluster.yaml
```

Comment:
Can you give an example without hardcoding the version? Might be annoying to keep this updated with new releases

Reply:
Yeah looks like that's possible.